src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -808,6 +810,21 @@ public function testGetByIdNotExisting()
     public function testPost()
     {
         $client = $this->createAuthenticatedClient();
+        /** @var TargetGroupRepositoryInterface $targetGroupRepository */
+        $targetGroupRepository = $this->getContainer()->get('sulu.repository.target_group');
```

Comment:
The bundle is optional, but I still have to test in here if it's working correctly in the combination.